TESTING.md

```diff
@@ -0,0 +1,67 @@
+# Testing the 'oneview' module
+The 'oneview' module provides a unit testing strategy and linting tools to help ensure its code style.
+This document will cover on how to execute and implement these tests.
+
+## Executing tests
+The tests can be executed by `rake` tasks and by their original tools, like `rspec`, `rubocop`, `puppet-lint` and `puppet parser`.
+
+### Rake
+All the test strategies and checks can be executed by the rake command.
+Please use `rake -T` to see a full list of commands.
+
+### Unit & Integration tests
```

Comment:
Integration tests? This section covers only the Unit tests.

Reply:
Added an 'and integration' to the section.